src/stories/LinkField.stories.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import ThemeProvider from '@mui/styles/ThemeProvider';
+import linnTheme from '../themes/linnTheme';
+import LinkField from '../components/LinkField';
+
+export default {
+    title: 'Components/LinkField',
+    decorators: [
+        story => (
+            <ThemeProvider theme={linnTheme}>
+                <div>{story()}</div>
+            </ThemeProvider>
+        )
+    ],
+    component: LinkField
+};
+
+export const LinkFieldExample = args => <LinkField {...args} />;
+
+LinkFieldExample.story = {
+    name: 'Standard link to match input field display'
+};
+
+LinkFieldExample.args = {
+    value: 'Test Link',
+    openLinksInNewTabs: false,
+    to: '/',
+    external: true,
+    disabled: false,
+    label: 'Test Link Example'
+};
```

Comment:
I admire your dilligence in adding the .stories file, but storybook is completely borked at the moment so sadly this will be neglected.
Doesn't do any harm to leave it in though

Reply:
i know. was just trying to be good